add more words to explain trackingSessionEvent in Flutter web. (#674)
Amplitude-Developer-Docs/amplitude-dev-center: docs/data/sdks/flutter/index.md

    | `setServerUrl()` | `String`. The API endpoint URL that events are sent to. Automatically selected by `ServerZone`. For example, `Amplitude.getInstance().setServerUrl(https://www.your-server-url.com)`. | `https://api2.amplitude.com/` |
    | `setUseDynamicConfig()` | `bool`. Find the best server url automatically based on users' geo location. For example, `setUseDynamicConfig(true)`. | `false` |
    | `setOptOut()` | `bool`. Opt the user out of tracking. For example, `Amplitude.getInstance().setOptOut(true)`.| `false` |

>     | `trackingSessionEvents()` | `bool`. Whether to automatically log start and end session events corresponding to the start and end of a user's session. For example, `Amplitude.getInstance().trackingSessionEvents(true)`. | `false` |

Reviewer: I think putting the exact Event name as it is sent from the SDK would help the user know what to look for in the event stream.

```suggestion
    | `trackingSessionEvents()` | `bool`. Whether to automatically log "[Amplitude] Session Start" and "[Amplitude] Session End" session events corresponding to the start and end of a user's session. For example, `Amplitude.getInstance().trackingSessionEvents(true)`. | `false` |
```

- `enableCoppaControl`
- `disableCoppaControl`

> - `trackingSessionEvents`. We are still tracking the sessionId in Flutter Web, but whether to automatically log start and end session events corresponding to the start and end of a user's session is not supported in Flutter Web.

Reviewer 2: Do we even fire the session events in Flutter web with JS SDK? Maybe we could claim we don't support session events in Flutter web? The current wording feels like this control still work for Flutter web.

> - `trackingSessionEvents`. We are still tracking the sessionId in Flutter Web, but whether to automatically log start and end session events corresponding to the start and end of a user's session is not supported in Flutter Web.

Reviewer: ```suggestion
- `trackingSessionEvents`. Flutter Web will still track the sessionId, but it is not possible to automatically log start and end session events corresponding to the start and end of a user's session.
```
